inventory/management/misc/batch_transfers.rst

```diff
@@ -0,0 +1,126 @@
+=======================
+Process Batch Transfers
+=======================
+
+Batch picking allows a single picker to handle a batch of orders,
+reducing the number of times he must visit the same location. In Odoo,
+it means you will regroup several transfers into the same batch
+transfer, then processing it via its form view or the barcode
+application.
+
+Create a Batch Transfer
+=======================
+
+To activate the batch picking option, go to :menuselection:`Inventory
+--> Configuration --> Configuration` and enable *Batch Pickings*.
+
+.. image:: media/batch_transfers_01.png
+    :align: center
+    :alt: View of the inventory settings. Process to enable the batch pickings in the Odoo Inventory app
+
+Then, go to :menuselection:`Inventory --> Operations --> Batch
+Transfers` and hit the create button.
+
+.. image:: media/batch_transfers_02.png
+    :align: center
+    :alt: View of the new menu, Batch Transfers, under operations
+
+Now, determine the batch transfer responsible and the type of transfers
+you want to include in the batch. To do so, click on *Add a line*.
+
+.. image:: media/batch_transfers_03.png
+    :align: center
+    :alt: View of a Batch Transfers form
+
+For example, use the filter feature to only have the *Pick* step
+before shipping the products to customers. After that, select the
+different transfers which should be included in the batch.
+
+.. image:: media/batch_transfers_04.png
+    :align: center
+    :alt: View of the list of transfers to choose for a single batch transfer and how to add them
+          to the batch transfer
+
+To see the products to pick for the different transfers, click on
+confirm. The document also shows the locations they have been reserved
+from if *Multi-locations* has been activated.
```

Comment:
```suggestion
To see the products to pick for the different transfers, click on
confirm*. The document also shows the locations from where they have been reserved
(if *Multi-locations* has been activated**).
```

* Isn't it Select? (if it follows the image above, because in the print the follows this paragraph there isn't a button 'Confirm' (thinking emoji))
** Is 'Multi_localtion' under Settings? When I read this I scrolled up to see that first print of the setting page to find this option, but I don't see it. So maybe tell the localization of this option, or link to a doc if there is one?) - suggestion from an illiterate person on the subject though. ^^

Reply:
Multi-location is a feature that is not mandatory to use batch transfers. So I didn't add a link to the doc. But if you think it can be useful, then I can add a note with the link to the doc :) 

About confirm, you're right. I'm too used to the "confirm" and "validate" buttons everywhere haha

Reply:
I think adding a note would be useful indeed 👍